Reviewed Space creation and Planning sections

Preeticp opened the pull request:
This PR majorly focuses on reducing the number of screenshots and trimming the Planner section as per discussions in #394 as well as addresses the comments [here](https://github.com/fabric8io/fabric8-online-docs/issues/388#issuecomment-398936784) about modifying the start text. 

Considering that this could be an individual standalone module, I have added the start text to this section itself. The next section already has the statement saying, if you closed the wizard earlier access it from...

mishaone commented on docs/topics/modules/creating_new_space.adoc:
```diff
@@ -17,7 +17,7 @@ endif::[]
 == Procedure
 
-Create a new space as follows:
+Create a new space in {osio} as follows:
```
Minor quibble, but is it necessary to mention osio here, since this is the middle of the module and the next line does mention osio again.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,65 +1,31 @@
 = Planning your development work
 
-After creating a space, use the {osio} planner to plan your development work and to ensure efficient execution of your project.
+When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow for launching and deploying a sample application before you launch the application.
```
The latter sentence is getting a bit clunky and therefore not as clear. Maybe something like this would be better: Close the wizard so that you can plan your development workflow before you launch the application wizard.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,65 +1,31 @@
-+
-NOTE: A scenario can have multiple realizing experiences and supporting value propositions.
-+
+.. Select *Experience* from the drop-down list, add the title *User should be able to launch applications through a simple work-flow*, and Press kbd:[Enter].
```
Suggest type rather than add when we are making the user type out strings.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -68,36 +34,21 @@ image::gs_feature_eg.png[Example Feature]
 * <<creating_che_workspace-hello-world,*Modify the quickstart codebase*>>
```
I know this is not part of your changes this time, but could we reduce these? Maybe to two tasks so that it's not a lot of administration for a relatively simple Getting Started task.

Preeticp replied:
Well Misha the whole point of having this section placed here and all the tasks here is to knit the disparate tasks in the getting started together. This section is what connects all the separate tasks we ask the user to perform. As such I would vote for retaining these tasks.

mishaone replied:
Yeah, this is probably entirely subjective.

I had an idea, though. Instead of just the two of us, we could open this up to a wider audience. I can take this whole "chapter", or at least a part of it, since it's a long chapter, to a minimalism session. Maybe 5.1-5.8 or so.  I can ask the writers discussing it to give us notes about what they suggest would improve the content and note it down and we can implement the changes. It'll be good to get some fresh eyes on the content anyway and I won't say anything about this particular module or anything, just let them look through the basics, flow, text, etc. and give us their thoughts. WDYT?

Preeticp replied:
> Yeah, this is probably entirely subjective.

@misha, trying to bring some objectivity to the discussion.
Question 1: what is our intent here?
1. If we want to simply showcase Planner usage, yes 1/2 tasks would be enough
2. If we want to use the Planner section to showcase the feature as well as tie the various seemingly disjoint actions in the Getting started experience together, I believe we need these 4 tasks.
I would suggest lets discuss the intent over a meeting. I am ok with whichever way we decide to go.

Question 2: Do we want to adress the length of the whole module or is it only the Tasks listed here?
If its the length, I have been toying with using the agile template instead of SDD, I have a feeling it could reduce the size a bit. However the number of tasks would be determined by our intent(1st question). If you are ok with this, I will try to have a rough agile draft in place for the meeting so we can evaluate.

On opening docs for minimalism review: +1 I have been thinking of sharing our docs in our minimalism sessions too, we record the sessions so we could simply share the recordings and work on them.
 
Also, please note, since yesterday, design discussions are happening to change the UX flow to make the whole experience cohesive. Should we wait till then? As depending on the changes a lot of things could become redundant. We simply may not need to have so many tasks as the experience may not seem disjoint then. Also, it might be better to have the writers have a look at it in the sessions once we update our docs as per the improved UX. Thoughts?

mishaone replied:
Re: intent - That's why it sticks out so much to me. I feel like the other modules around it are really specific and limited by design so that the user goes through a relatively simple workflow while seeing where to find various features and how to use them. To that end, just having a single work item in the planner to edit the code is probably more than enough, since it illustrates how to use the planner sufficiently as a whole. I think adding multiple work items here complicates it a bit too much when the actual code changes we do later are just copy and pasting code lines or just minor edits. Does that make sense? I think trimming it all down to only the basics for a new user to know would be in line with this document. The details are best left to the user guide.

Q2: As I said, it's just a matter of how much we can look through in about an hour. If there's a lot of feedback, we may only make it through maybe 6-7 modules. If less feedback, we might manage the whole ch 5. I'm open to either. It's important to start from the beginning of the chapter so that the intent and context is present as we go through each module step.

I don't have any issues with the length, please use whichever option you think best fits the workflow, since you're the planner expert. :)

Re: updates. I think there are literally always going to be big updates for a product this young. The minimalism review isn't a judgment on the quality of our docs or how accurate it is to the product, just a bunch of other writers having a first look at something and pointing out bits that could be changed to improve things like the flow of the document, make things obvious if they aren't, enhance the format or structure, etc. Having that sort of feedback is useful at any time so I would recommend we just jump into it as soon as we finish our reviews. It's a bit trying to listen to critique of one's own work so if I submit something, I usually just sit in a corner without argument and listen to the discussion and feedback and note it down. Then, we can discuss the feedback and see how useful it is to us and apply ti accordingly. What do you think?

Preeticp replied:
Re: Intent, we have a difference of opinion here, let's hash it out in a meeting. Each of the tasks relate to the actions the user has to perform as part of the getting started.
Re: Length, based on the time I have at hand, I will experiment with the agile template and see if we can arrive at a shorter version.
Re: Minimalism, I understand the limitation of time and sections we can review within the hour. I have been part of these sessions so I know what you mean about writers providing their feedback. I suppose we would be in different minimalism sessions hence it would be good to share the recordings is what I suggested.

mishaone replied:
I'm reluctant to have meetings about differing opinions because they're just that: opinions. It's easier to get a third party's opinion instead and then we have a clear way forward. However, happy to meet to hash this out later. I'll be online later (will message time) and can meet before that time.

Preeticp replied:
Agreed, I was suggesting hashing out the intent within the team in our weekly meeting when @rkratky would be available too. Again the group review would add to our input. 
I do not wish to impose on your personal time. Let's discuss in the Tuesday team meeting.

rkratky replied:
Quick notes:

- Agree w/ Misha re the extra WIs. I don't think they help to tie stuff together. Instead, it looks like a homework -- we're not teaching the user how to create a WI, instead, we're telling them "create a whole bunch of them because practice makes perfect". If I were the user, I would snicker and ignore it. We need to be mindful that users don't read our docs because they thirst for knowledge; they read them with disdain when all else failed.
- The whole 5.2.2 section is too long. The list of steps has 8 items and some of them multiple sub-items. Too long.
- Headings should consistently use gerund verb forms.

Preeticp replied:
> create a whole bunch of them because practice makes perfect

That was not the intention at all, the intention was to create tasks for each stage of the getting started.
If this is the implication, I will remove the extra WIs. Definitely don't need any snickering and ignoring directed at our docs :)
Re: length, SDD makes the process long. I am working on an Agile workflow, hopefully that would pare it down.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -68,36 +34,21 @@ image::gs_feature_eg.png[Example Feature]
 image::gs_tasks_egs.png[Example Tasks]
 
-. Similarly, for the *Fundamental* - *The application should be secure with appropriate licenses*, create a child experience*, feature, and task . The task could be titled <<analytics_stack_report,*Use analytics to identify and resolve security and license issues in the application*>>.
+. Similarly, select *Fundamental* in the *Add Work Item* pane, to create a fundamental work item with the title *The application should be secure with appropriate licenses* and use the in-line add option to create a child experience, feature, and task for it. The task can be titled <<analytics_stack_report,*Use analytics to identify and resolve security and license issues in the application*>>.
```
Similarly is one of the words we should not use, unless it's clear what we're comparing it. In this case, I'd suggest not using it, since it's not really necessary anyway.

s/The task can be titled/Title the task or something else. We should avoid can/may/might/should/etc. and ensure the language is directive.

mishaone replied:
* Unless we are actually talking about one of many options. In this case, I believe the usage is meant to be directive, i.e. do this rather than you can do this if you want to.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -68,36 +34,21 @@ image::gs_feature_eg.png[Example Feature]
+image::gs_modify_wi.png[Modify work item]
 
-You have now learned how to use the Scenario Driven Development planning methodology to plan your work by creating work items and to track and execute them by associating them to iterative development cycles.
+You can now click *Test Iteration* to see the work items associated with it and use the planner to track your development work for the iteration.
```
s/You can now click/Click

Preeticp commented:
Thanks @mishaone will wok on this.

Preeticp commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,65 +1,31 @@
+When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow for launching and deploying a sample application before you launch the application.
 
-NOTE: The development process you select while creating the space determines the planner's guided work item type hierarchy.
+{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the planner's guided work item type hierarchy.
```
@mishaone Re: Capitalization we agreed to use small case for planner/analytics instead of Title case as these are not exactly referenced in the UX. I remember us changing the title cases to small earlier after discussing this.

Preeticp replied:
Fixed the possessive.

Preeticp commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,65 +1,31 @@
 .. In the *Add Work Item* pane, select *Scenario* from the drop-down list.
 +
-image::quickadd_wi_type.png[Quick Add work item type]
+image::quickadd_scenario.png[Quick add scenario]
```
@mishaone I think we should include this screenshot as there are many things I am trying to show in this screenshot:
1. It is the first screenshot which introduces the user to Planner in the GS
2. Depicts the flow from Plan>Backlog>Scenario>Quick Add
3. Depicts the difference between the Quick Add and the Inline add
4. I removed the screenshot for the step to create a fundamental work item, so the user can refer to this screenshot and draw inference for that too.

Preeticp commented:
@mishaone I have fixed most of the comments, except for a couple. I have added my views for the same, let me know if a quick Bjn to discuss and settle it works for you.

mishaone commented:
I should have said this already but absolutely okay to merge for now. I don't want to hold up your work but I think a neutral audience assessing all the content at some point would be a good idea.

Preeticp commented:
You are not holding up my work, I am working on other things. In fact I appreciate your feedback, such discussions add great value and help us make our docs better. So though I don't want to hold of on merging forever I would rather we merge in something good and of use to the end user.

Preeticp commented:
@mishaone @rkratky used the Agile template to modify the Planning module. Please review.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
+{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the guided work item type hierarchy in the planner.
 
-This example walks you through the Scenario Driven Development planning process to plan your work.
+This example walks you through the Agile planning process to plan your work.
```
"walks you through" seems a little informal for tech communication to me. I'd suggest something like "This is an example of planning your work using the Agile planning process".

Preeticp replied:
+1 Will change

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
-.. In the *Work Item Title* field, add *Enable an end to end, developer work-flow, using the continuous integration and deployment framework* as the title.
-.. Press kbd:[Enter] to create the scenario.
+. In the dashboard of your space, select the *Plan* tab at the top of the page. The default *Backlog* view displays the *Work Items* for your space.
+. Select the *Show Tree* check box to use the quick-add and in-line add options to quickly create workitems and their children.
```
I was just testing these instructions and this was selected by default for me (fresh env, not used planner before here).

Preeticp replied:
Are you sure? Cause for me it did not, I checked yesterday, will check again.

mishaone replied:
Yes, sorry, see above, I was in a SDD space. I'm now in an Agile one and I can see the options.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
-.. Press kbd:[Enter] to create the scenario.
+. In the dashboard of your space, select the *Plan* tab at the top of the page. The default *Backlog* view displays the *Work Items* for your space.
+. Select the *Show Tree* check box to use the quick-add and in-line add options to quickly create workitems and their children.
+. In the *Add Work Item* pane, select *Theme* from the drop-down list, type *Enable a complete developer work-flow in a continuous integration and delivery framework* as the title, and press kbd:[Enter] to create the theme.
```
Sorry, one last odd thing: "to create a theme". What is this theme? We haven't introduced this idea and so far we've been talking about adding or creating work items.

Preeticp replied:
@mishaone Agile concepts have been added to the glossary. There is another PR to fix some review comments from Robert and will be merged as soon as he approves it.

Preeticp replied:
These are not flat work items like issues in GH.

mishaone replied:
My example was about JIRA, which should be a closer comparison, not GH. JIRA issues can have types, subtasks, etc. but are still called issues, not any of the types.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
 
-. Create a child *Experience* for the *Scenario* using the in-line add option as follows:
-.. In the work item list, click the (+) icon adjacent to the scenario to see the in-line *Add Child Work Item* pane below it.
+. Create a child *Epic* for the *Theme* as follows:
```
Again, here, the terminology is very confusing, IMO. Why are we calling this a child epic instead of a child work item? Also, child X implies that there is a parent X, but for some reason we're calling parent work items themes and child work items child epics. This is going to confuse users, I think.

Preeticp replied:
@mishaone have you selected Agile as the development process? All the WITs you are quoting are from SDD. Please test using an Agile space.

mishaone replied:
Ah, right. Thanks, I can see the Theme and Epic now, but would still call them Work Items and Child Work Items for clarity and because the interface calls them the same for clarity as well. Referring to them by type will be confusing, like referring to JIRA issues as Features, Bugs, Tasks, etc.

Preeticp replied:
@mishaone Calling them workitem and child work item, imo increases ambiguity, user will wonder which child wit is being referred to, Child epic, grandchild story or great grandchild task. Also the realtion between the various types will never be clear.

mishaone replied:
Honestly, I'm not sure why we need grandchild stories to begin with in a simple hello world application example. Maybe we should have a team meeting to discuss what exactly you want to show users in this example and whether all of it is relevant for a hello world example? I suspect simplifying things is better with this kind of use case.

Preeticp replied:
Exactly, this is what I suggested when I said let's meet to discuss the intent. We started off wanting to tie in the various seemingly disjoint actions in the GS as the intent, now we seem to be veering to having a single work item. 
A lot of discussions happened and efforts went in before this was pushed in the first time, both in google doc draft as well as the PR. But now we seem to want to change the intent. All good, except that we need to decide on how we move forward.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
-. Create a child *Experience* for the *Scenario* using the in-line add option as follows:
-.. In the work item list, click the (+) icon adjacent to the scenario to see the in-line *Add Child Work Item* pane below it.
+. Create a child *Epic* for the *Theme* as follows:
+.. In the work item list, click the (+) icon adjacent to the theme to see the in-line *Add Child Work Item* pane below it.
```
Reiterating the above: here we (and the UI) calls it a child work item. Let's just stick to the terms work item and child work item to reduce confusion?

mishaone replied:
I'm a bit confused now. Your linked document suggests that there are three work item types: Work Items (having a work item as a work item type is a bit odd in itself, but ignoring this for now), Tasks, and Iterations. I'm suggesting we refer to these (Work Items) as Work Items rather than as Themes, Epics, etc.

Preeticp replied:
@mishaone the linked doc merely gives definitions of the WITs in Agile, not necessarily in order. To map to SDD, Themes would map to Scenarios, Epics map to Experiences, and Story to Requirements. Tasks are children of stories.
As I explained earlier, calling them work items might confuse the user more, we are expecting the user to know codebases and pipelines. Themes, epics anduser stories are common agile terms used in most enterprises and we can expect most users to be familiar with them. In addition we would have infotips to explain them to the users.

mishaone replied:
Yes, agreed, BUT these are just work items. We can call them Theme type Work Items, which is clearer because we are still talking about Work Items. I would suggest removing the problem and just simplifying the use case for tutorials because the idea is to introduce people to the basics, not overwhelm them with terminology. Bear in mind this is the first thing people are doing after creating a space so if this is too confusing, they may not want to continue. Simple and clear would be my recommendation here.

Preeticp replied:
Lets discuss this Misha, depending on the final intent and the task we want to highlight we can move this module somewhere down the guide. We dont break the flow too this way.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
 +
 image::gs_inline-add_eg.png[Inline Add]
 +
+.. Select *Epic* from the drop-down list, type the title *User should be able to launch applications through a simple work-flow*, and Press kbd:[Enter].
```
Suggested: User can launch applications

I'm not sure what the title means, though. What task is this tracking? We don't do anything in the code to change how users launch applications.

Preeticp replied:
@mishaone I really feel we need a discussion on Planner and how it works, this is exactly what I was trying to tell you all yesterday. The tracking part can actually be only through tasks which is why we had 4 tasks. Rest all are just parent child work items meant for planning. Stories and tasks are execution level. Please see this: https://docs.google.com/document/d/1-Zccq97I6IhVGkSLaO_PWokyuW63fBJFQFNVZn64ucw/edit?usp=sharing

mishaone replied:
I absolutely understand what you're saying (and think that's good material for something like the User Guide), but I'm not sure why we can't just have one item, let's say a Task type Work Item, that tracks a single task we will do in the hello world example? That would seem to provide the information users need for now, while the UG can have details about these and how to use them for later.

Preeticp replied:
Will work on the titles, the titles need to highlight the interconnections of a parent child relation, so they have to be similar while being broader or narrower in their focus for Parent and child WITs respectively.

mishaone replied:
I'm not sure you need to explicitly state how they are related in the titles, Preeti. They're a parent and child, which implies that the latter is a subset of the former.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
-* <<approving_build_pipeline,*Test the application in Stage and approve to the Run environment*>>
-* <<creating_che_workspace-hello-world,*Modify the quickstart codebase*>>
-* <<reviewing_publishing_changes-hello-world,*Review and publish changes to your codebase*>>
+. Using the in-line add option create an execution oriented child *Story* for the *Epic* with the title *User should be able to easily launch a sample Vert.x application* and a child *Task* for the story titled *Review and publish changes to your codebase*.
```
This is IMO very confusing. If it's necessary to create another work item, we should have specific steps for this part. I clicked what I assume is the in-line option but it has both Story and Epic types in the drop down menu and it's not clear which one to select or why, the title should be shorter and to the point. This reads like several steps mashed together at the moment.

Preeticp replied:
Yes I clubbed 2 separate steps here since they were to be done in a similar manner as highlighted with screenshots in the previous step. The other reason was to cut down on the number of steps. The inline option does not have both options, please see the screenshot for the inline add. It was precisely for this reason I retained substeps for inline add as well as the screenshot as I saw potential for confusion. Have you tried rendering this doc and then reviewed it. I hope along with the screenshot this is easier to follow.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
-* link:user-guide.html#associating_work_items_with_an_iteration-user-guide_iterations[Associate the work items to the *Test Iteration*]
-+
-See link:user-guide.html#working_with_work_items[working with work items] for detailed information about modifying and tracking work items by associating them to iterative development cycles.
+. In the left panel, click the (+) icon adjacent to *Iterations* to link:user-guide.html#creating_a_new_iteration[create an iteration] called *Test Iteration*, force activate it, and  link:user-guide.html#associating_work_items_with_an_iteration-user-guide_iterations[associate the story and task to the *Test Iteration*].
```
This should also be broken down into multiple steps that are easy to follow along. I'm not sure we need an iteration for this example to start with, TBH. I don't know why we're directing users to a separate document for associating the story and task, this should be elaborated here as well.

Preeticp replied:
@mishaone this was the case earlier, but it adds to the steps and substeps. This was changed based on @rkratky  inputs that this module is very long, we had 8 steps with many of the steps having substeps. Let me know if we want substeps again
Well Iteration is the way you will track the WIs in the development cycle. 
There was a screenshot along with expalnation for the same, I removed it since we wanted to cut down on screenshots and also number of steps. Please see the current prod version of our docs. 
Seems like I am going back and forth on these. I suggest we all meet sometime to discuss exactly how we want to go about this module. I will note down the AIs from there and work on it.

mishaone replied:
I don't think the best way to address longer modules is to skip or merge steps, tbh. Maybe we can reduce the things we're doing so that it's simpler and shorter instead somehow?

Preeticp replied:
>I don't think the best way to address longer modules is to skip or merge steps, tbh.
I agree, but with the current intent and the feedback to cut down length this and links to the detailed docs was what I thought of.
I am open to suggestions, we have our team meeting on Tuesday, I suggest we discuss then or if we can have it earlier that would be great too. Discussing this here is really not working out.

mishaone replied:
I don't honestly know that we can be much help given that you're the "expert" on planner so to speak. I think the way forward is to identify a more streamlined and minimalist sort of workflow that suits the context here. I don't think there's much I can say about the specifics of that since I don't know much about the intended workflow you have at the moment or what the best thing to show users at this point is. Maybe Robert can say something about that?

Preeticp replied:
My expertise or lack of it is not the point, this one is for our docs team to decide on. @mishaone my aim for the discussion would be 1. finalize on the intent: Tying disjoint actions or not, 2. I could explain the 2 workflows we can have, minimal or slightly deeper. I understand your viewpoint, I started with a different one. I just want a clarity on which one we as a team want to follow. I am happy to go along what ever all of us decide.
I don't see why we are so hesitant on meeting on this. We are spending a fair bit of time writing back and forth for what could be resolved in 1 meeting. AFAIR a meeting was able to resolve 2 issues we were stuck on earlier fairly quickly.

mishaone replied:
I mean, once again we're at a crossroads with two opinions. I think this can be relatively easily and quickly decided on via email or comments. I thought we had already decided on a more streamlined and basic workflow for this document. I don't know much about the planner, as I said, so I'm not sure what I can contribute to this aside from what I've already contributed. I'm happy to meet but unsure of the benefits in this particular instance.

mishaone commented on docs/topics/modules/planning_development_work.adoc:
```diff
@@ -1,54 +1,29 @@
+image::gs_associate_wi_with_iteration.png[Associate work item with an iteration]
 
-You can now click *Test Iteration* to see the work items associated with it and use the planner to track your development work for the iteration.
+You can now use the *Test Iteration* in the left panel to see the work items associated with it and track your work.
```
One thing you probably already have on your radar for later, but we should have some instructions later when we do something about returning to the planner and updating the relevant work items.

Preeticp replied:
This was already in place for the earlier version, what Inow need to do is depending on how this module goes modify or delete existing stuff.

rkratky commented:
Hi @Preeticp, @mishaone. So far, this PR generated over 70 notification e-mails. I'm not saying that's wrong, but I want you to put things in perspective. Is the issue really worth that much debate?

I have not read this entire discussion, and I am not familiar with the Planner app to make an informed judgement. So, the following a is a generalized comment.

I would like to GSG to only contain essential content. That's highly subjective metrics, but let me try to provide an example:

It's Friday 15.30, and the office is unbearably hot. Outside temp is some 40 deg. C, inside is a lovely 36, according to the remote-control panel of the non-functioning AC. I have just come out of a computer lab (no windows, no AC) with 15 running workstations occupied by 15 sweaty guys. We were there for over 4 hours, taking an exam. I'm exhausted. Essential content is such that I would be willing to read now and still grudgingly acknowledge that it really is something I needed to know :)

Preeticp commented:
Post discussion here is the summary: We will have a minimal doc for the Planner in the Getting started with links to User guide which has detailed information. 
We could have an article kind of doc for planner which could create a scaffolding for the Getting started and explain the Planner in more detail. The difference between User guide and this would be the tone of voice and a guided path to go from A to Z. 
@mishaone, can we use the current mini assembly you have created for this? Currently it is the same as the GSG module, but we can replace it with a more article based Planning module as discussed in the meeting today.
@mishaone and @rkratky please note I am closing this PR and will raise a fresh one based on the above discussion.
